controllers/entities.js

```diff
@@ -30,9 +30,10 @@ exports.create = function(req, res, next) {
     if (result.errors > 0) return next(result.first_error);
     return result.changes[0].new_val;
   })
-  .then(entity =>
-    fanout.resolvePermissions(entity.id, req.body.permissions)
-    .then(() => res.send(entity)))
+  .then(entity => fanout.resolvePermissions(entity.id, req.body.permissions)
+    .then(() => r.table('entities').get(entity.id))
+    .then(entity => res.send(entity))
+  )
   .then(next)
```

Comment:
Yeah you're right in that next gets called with the return value of `res.send()`, though `res.send()` returns the response context which is fine to pass through next, so this pattern works fine, ie: `next(res.send('hi mum'))` will go to the next handler just as `next()` would. 

Doesn't hurt to be explicit though, so i'll update it.